packages/@aws-cdk/aws-elasticloadbalancing/test/loadbalancer.test.ts

```diff
@@ -89,6 +89,50 @@ describe('tests', () => {
     });
   });
 
+  test('add an Instance as a load balancing target', () => {
+    // GIVEN
+    const stack = new Stack();
+    const vpc = new Vpc(stack, 'VCP');
+    const instance = new Instance(stack, 'targetInstance', {
+      vpc: vpc,
+      instanceType: InstanceType.of( // t2.micro has free tier usage in aws
+        InstanceClass.T2,
+        InstanceSize.MICRO,
+      ),
+      machineImage: MachineImage.latestAmazonLinux({
+        generation: AmazonLinuxGeneration.AMAZON_LINUX_2,
+      }),
+    });
+    const elb = new LoadBalancer(stack, 'LB', {
+      vpc,
+    });
+
+    // WHEN
+    elb.addListener({ externalPort: 80, internalPort: 8080 });
+    elb.addTarget(new InstanceTarget(instance));
```

Comment:
Can we also do a test where these are reversed to make sure we still see the security group rules?